modules/acm/src/main/java/edu/gemini/epics/acm/CaSimpleObserveSenderImpl.java

```diff
@@ -192,17 +211,17 @@ public synchronized CaCommandMonitor post() {
         if (!currentState.equals(idleState)) {
             failCommand(cm, new CaCommandInProgress());
         } else {
-            currentState = new CaSimpleObserveSenderImpl.WaitApplyPreset(cm);
 
             try {
                 apply.setDir(CadDirective.START);
+                currentState = new CaSimpleObserveSenderImpl.WaitApplyPreset(cm);
+                if (timeout > 0) {
+                    timeoutFuture = executor.schedule(() ->  CaSimpleObserveSenderImpl.this.onTimeout(), timeout, timeoutUnit);
+                }
```

Comment:
This also looks familiar.

Reply:
It is the same as another change but I don't think we should refactor this one out, It may be confusing as I want to make both `*observerSender*` classes indepenent